en/flight_modes/follow_me.md

```diff
@@ -8,16 +8,16 @@ author: Jimmy Johnson
 
 *Follow Me* mode allows a multicopter to autonomously follow and track another system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.

```

Comment:
Is it still just multicopter that supports this mode?

Reply:
Yep, only multicopter